docs/guides/universal-profile/check-if-address-is-universal-profile.md

```diff
@@ -24,95 +59,224 @@ if (!isAddress(address)) {
+import UniversalProfile from '@lukso/lsp-smart-contracts/artifacts/UniversalProfile.json';
+import { SupportedStandards } from '@lukso/lsp-smart-contracts/constants';
+import { ethers } from 'ethers';
+
+const provider = new ethers.JsonRpcProvider('https://rpc.l14.lukso.network');
 
-const contract = new web3.Contract(getDataABI, address);
+const universalProfileAddress = '0x...'; // The address of the contract that you are verifying
+const unviersalProfile = new ethers.Contract(universalProfileAddress, UniversalProfile.abi, provider);
 
-const supportedStandard = await contract.methods['getData(bytes32)'](SupportedStandards.LSP3UniversalProfile.key).call();
+const supportedStandard = await unviersalProfile['getData(bytes32)'](SupportedStandards.LSP3UniversalProfile.key);
 
 if (supportedStandard !== SupportedStandards.LSP3UniversalProfile.value) {
   throw new Error('Address does not support LSP3UniversalProfile standard');
 }
 ```
 
 <!-- prettier-ignore-end -->
+
+  </TabItem>
+
+</Tabs>
+
+## Final Code
+
+<Tabs>
+  
+  <TabItem value="web3js" label="web3.js">
+
+<!-- prettier-ignore-start -->
+
+```javascript
+import UniversalProfile from '@lukso/lsp-smart-contracts/artifacts/UniversalProfile.json';
+import { INTERFACE_IDS, SupportedStandards } from '@lukso/lsp-smart-contracts/constants';
+import Web3 from 'web3';
+
+const web3 = new Web3('https://rpc.l16.lukso.network');
+
+if (!web3.utils.isAddress(address)) {
+  throw new Error('Invalid address');
+}
+
+// We assume that the contract is a Universal Profile
+const universalProfileAddress = '0x...'; // The address of the contract that you are verifying
+const unviersalProfile = new web3.eth.Contract(UniversalProfile.abi, universalProfileAddress);
+
+const supportsLSP0Interface = await universalProfile.methods.supportsInterface(INTERFACE_IDS.LSP0ERC725Account).call();
+// true or false -> if false, this address is not a Universal Profile.
+if (!supportsLSP0Interface) {
+  throw new Error('Contract does not support LSP0ERC725Account interface');
+}
+
+const supportedStandard = await unviersalProfile.methods['getData(bytes32)'](SupportedStandards.LSP3UniversalProfile.key).call();
+
+if (supportedStandard !== SupportedStandards.LSP3UniversalProfile.value) {
+  throw new Error('Address does not support LSP3UniversalProfile standard');
+}
+```
+
+<!-- prettier-ignore-end -->
+
+  </TabItem>
+
+  <TabItem value="ethersjs" label="ethers.js">
+
+<!-- prettier-ignore-start -->
+
+```javascript
+import UniversalProfile from '@lukso/lsp-smart-contracts/artifacts/UniversalProfile.json';
+import { INTERFACE_IDS, SupportedStandards } from '@lukso/lsp-smart-contracts/constants';
+import { isAddress, ethers } from 'ethers';
+
+const provider = new ethers.JsonRpcProvider('https://rpc.l14.lukso.network');
+
+if (!isAddress(address)) {
+  throw new Error('Invalid address');
+}
+
+// We assume that the contract is a Universal Profile
```

Comment:
here too, we are specifically not assuming its a UP as we are checking its interface